custom_components/ha_kia_hyundai/const.py

```diff
@@ -20,9 +20,29 @@
     Platform.LOCK,
     Platform.NUMBER,
     Platform.SENSOR,
```

Comment:
move up a line to alpha sort (sorry, I know this feels picky, but I've been working for weeks in another integration to fix a load order issue that only showed up when this array was alpha sorted)